From 5196fd68ecb5c9e0b4eb9b6fb0901d78fdbe400b Mon Sep 17 00:00:00 2001
From: Ankit Agarwal <ankit.agarwal@intel.com>
Date: Fri, 21 Jun 2024 14:28:55 +0530
Subject: [PATCH] [WA] Disabled wi-fi display service.

Wireless display depends on wi-fi. Since wi-fi module is not up,
So disabling it for now.

Tests: Prepared EB and device is booting fine.

Tracked-On: NA
Change-Id: I96d5a08f517b776ab3086e35d78b343a1d325787
Signed-off-by: Ankit Agarwal <ankit.agarwal@intel.com>
---
 overlay/frameworks/base/core/res/res/values/config.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/overlay/frameworks/base/core/res/res/values/config.xml b/overlay/frameworks/base/core/res/res/values/config.xml
index e748928..faf96b1 100644
--- a/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/overlay/frameworks/base/core/res/res/values/config.xml
@@ -70,7 +70,7 @@
          * The remote submix module "audio.r_submix.default" must be installed on the device.
          * The device must be provisioned with HDCP keys (for protected content).
     -->
-    <bool name="config_enableWifiDisplay">true</bool>
+    <bool name="config_enableWifiDisplay">false</bool>
 
     <!-- Set to true if the wifi display supports compositing content stored in gralloc
          protected buffers. For this to be true, there must exist a protected hardware path
-- 
2.34.1

